.github/workflows/startManualRelease.yml

```diff
@@ -0,0 +1,37 @@
+# Trigger a manual version bump and github release
+name: Start Manual Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18.11.x'
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.IDEE_GH_TOKEN }}
+      - name: Conventional Changelog Action
+        id: changelog
+        uses: TriPSs/conventional-changelog-action@d360fad3a42feca6462f72c97c165d60a02d4bf2
+        # Get the version bump from Changelog Action + Create Tag
+        with:
+          git-user-name: Release Bot
+          git-user-email: ${{ secrets.IDEE_GH_EMAIL }}
+          github-token: ${{ secrets.IDEE_GH_TOKEN }}
+          skip-version-file: true
+          skip-on-empty: false
+          output-file: false
+          pre-commit: ./scripts/bump-lerna-version.js
```

Comment:
hummmmm insteresting.  So the conventional-changelog is also going to try and bump the version in package.json.  Does skip-version-file take care of that for us I guess?  Does bump-lerna-version also do a commit to commit all the packge.*json updates?

I'm wondering if we just need to do the version bump in learn as a separate step that happens before (or maybe even instead of) the change log thing.

Reply:
So the reason I like to have it here is so the math is the same for the version + the git tag. Conventional changelog decides what the version should be and passes it to the bump-lerna-version script for us to read and just set, rather than having to decide that separately. We don't seem to have any option to set the version into conventional changelog... so this is the way.

Reply:
gotcha, makes sense.  I was just looking at the error and can't understand why it's beefing about the export in that preCommit script.